components/modal/modal.js

```diff
@@ -9,6 +9,7 @@ import { getClassNamesWithMods } from '../_helpers';
  */
 class Modal extends Component {
   state = {
+    isActive: false,
```

Comment:
Although it's fine, again, the method `componentDidMount` check for that property and call the `open` method.